tools/actions_local_runner.py

```diff
@@ -49,27 +97,30 @@ async def run_step(step, job_name):
             stderr=subprocess.PIPE,
         )
 
-        stdout, stderr = await proc.communicate()
-        cprint(col.BLUE, f'{job_name}: {step["name"]}')
+        stdout_bytes, stderr_bytes = await proc.communicate()
+
+        header(proc)
     except Exception as e:
-        cprint(col.BLUE, f'{job_name}: {step["name"]}')
+        header(proc)
         print(e)
```

Comment:
This would fix an error @samestep encountered (i.e. stdout_bytes might not be defined if `proc.communicate()` raised an exception
```suggestion
        print(e)
        return False
```

Reply:
To avoid these sorts of simple errors in the future, @driazati could you add a `tools/test/test_actions_local_runner.py`?